akka-docs/src/main/paradox/stream/operators/StreamConverters/fromInputStream.md

```diff
@@ -14,4 +14,32 @@ Create a source that wraps an `InputStream`.
 
 ## Description
 
-TODO: We would welcome help on contributing descriptions and examples, see: https://github.com/akka/akka/issues/25646
+Creates a Source from a `java.io.InputStream` created by the given function.  Emitted elements are up to `chunkSize` 
+sized `akka.util.ByteString` elements.  The actual size of emitted elements depends how much data the 
+underlying  `java.io.InputStream` returns on each read invocation. Such chunks will  never be larger 
+than chunkSize though.
+
+You can configure the default dispatcher for this Source by changing 
+the `akka.stream.materializer.blocking-io-dispatcher` or set it for a given Source by 
+using `akka.stream.ActorAttributes`.
+
+It materializes a @java[`CompletionStage`]@scala[`Future`] of `IOResult` containing the number of bytes read from the source file 
+upon completion,  and a possible exception if IO operation was not completed successfully. Note that bytes having
+been read by the source does not give any guarantee that the bytes were seen by downstream stages.
+
+The created `InputStream` will be closed when the `Source` is cancelled.
+
+See also @ref:[fromOutputStream](fromOutputStream.md)
+
+
+## Example
+
+Here is an example using both `fromInputStream` and `fromOutputStream` to read from a stream, uppercase the read 
+content and write back out into another stream.
```

Comment:
maybe clarify InputStream and OutputStream since those are not Akka Streams
"read from an `InputStream`", "out into another `OutputStream`"